x-pack/plugins/infra/public/components/asset_details/tabs/common/popover.tsx

```diff
@@ -28,16 +27,13 @@ export const Popover = ({
     <EuiPopover
       panelPaddingSize="s"
       button={
-        <EuiIcon
-          data-test-subj={props['data-test-subj']}
-          type={icon}
-          color={iconColor ?? 'text'}
-          size={iconSize ?? 'original'}
-          onClick={togglePopover}
-          css={css`
-            cursor: pointer;
-          `}
-        />
+        <button onClick={togglePopover} data-test-subj={props['data-test-subj']}>
```

Comment:
A simple HTML button turned out to be a better choice because we don't have to override any EUI styles.

Reply:
Isn't it something we should highlight to the `EUI` team instead?

Reply:
One could simply override the `transform`, `transition`, and a few other CSS attributes from the `EuiButtonIcon` component. It would make it a simple button like this one. I don't think it's something EUI doesn't allow you to do. It's more about using a more straightforward implementation to achieve the same thing. instead of this:

```tsx
<EuiButtonIcon
          data-test-subj={props['data-test-subj']}
          iconType={icon}
          color={iconColor ?? 'text'}
          size={iconSize ?? 'xs'}
          onClick={togglePopover}
          css={css`
            inline-size: auto;
            block-size: auto;
            transition: none;
            transform: none !important;
          `}
        />
 ```